doc/develop/api/api_lifecycle.rst

```diff
@@ -94,20 +94,20 @@ In order to declare an API ``stable``, the following steps need to be followed:
    `Zephyr Architecture meeting`_ where, barring any objections, the Pull Request
    will be merged
 
-.. _stable_api_changes:
+.. _breaking_api_changes:
 
-Introducing incompatible changes
-================================
+Introducing breaking changes
+============================
 
 A stable API, as described above strives to remain backwards-compatible through
```

Comment:
Very much a nit as this is not related with the initial scope of this PR at all, but if you feel like also addressing some minor grammar tweaks (there might be other missing commas, just had a quick look, feels like it would be worth doing a full pass).

```suggestion
A stable API, as described above, strives to remain backwards-compatible through
```